### PR TITLE
DOCSP-33835 100.9.1 release notes

### PR DESCRIPTION
**Pull request by ltran-mdb2**
## DESCRIPTION

100.9.1 database tools release notes

## STAGING

https://preview-mongodbltranmdb2.gatsbyjs.io/database-tools/DOCSP-33835/release-notes/database-tools-changelog/

## JIRA

https://jira.mongodb.org/browse/DOCSP-33835

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=654e8d465b686d6bd92f9d24

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

**Review by ianf-mongodb — APPROVED** [review]
LGTM

File: source/includes/changelogs/releases/100.9.1.rst [hunk 1]
```diff
@@ -0,0 +1,16 @@
+.. _100.9.1-changelog:
+
+100.9.1 Changelog
+-----------------
+
+*Released 2023-11-09*
+
+- This version updates external dependencies and fixes an issue where
+  mongodump could change the ordering of fields in a view definition.
```
Comment (sarah-olson-mongodb) [inline source/includes/changelogs/releases/100.9.1.rst:9]:
```suggestion
This version updates external dependencies and fixes an issue where
  mongodump could change the ordering of fields in a view definition.
```

**Review by sarah-olson-mongodb — COMMENTED** [review]
Thanks so much @ltran-mdb2. This is minor, but would you mind making the release description regular paragraph text, rather than a single list item.

File: source/includes/changelogs/releases/100.9.1.rst (hunk 1 above)
Comment (ltran-mdb2) [reply source/includes/changelogs/releases/100.9.1.rst]: Done!

**Pull request merged**